From 09f90007cb4b931ac060d9cc3792bb7466c8243e Mon Sep 17 00:00:00 2001
From: Anil Kumar Mamidala <anil.mamidala@xilinx.com>
Date: Wed, 17 Mar 2021 10:39:20 +0530
Subject: [PATCH 1782/1851] media: i2c: ap1302: Update yuv420 to xilinx NV12
 media bus format

commit 3e650ef54193239746604beb4ba79417b022a805 from
https://github.com/Xilinx/linux-xlnx.git

Update yuv420 format to xilinx NV12 media bus format of VYYUYY8_1X24
instead of UYYVYY8_0_5X24.

Reported-by: Chris Kohn <christian.kohn@xilinx.com>
Signed-off-by: Anil Kumar Mamidala <anil.mamidala@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/i2c/ap1302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ap1302.c b/drivers/media/i2c/ap1302.c
index 13e7cf513292..16616abe8bcd 100644
--- a/drivers/media/i2c/ap1302.c
+++ b/drivers/media/i2c/ap1302.c
@@ -438,7 +438,7 @@ static const struct ap1302_format_info supported_video_formats[] = {
 		.out_fmt = AP1302_PREVIEW_OUT_FMT_FT_YUV_JFIF
 			 | AP1302_PREVIEW_OUT_FMT_FST_YUV_420,
 	}, {
-		.code = MEDIA_BUS_FMT_UYYVYY8_0_5X24,
+		.code = MEDIA_BUS_FMT_VYYUYY8_1X24,
 		.out_fmt = AP1302_PREVIEW_OUT_FMT_FT_YUV_JFIF
 			 | AP1302_PREVIEW_OUT_FMT_FST_YUV_420,
 	},
-- 
2.31.1

